.gitignore

```
@@ -12,6 +12,7 @@ db.sqlite3-journal
 media
 locallibrary/__pycache__/
 catalog/__pycache__/
+catalog/migrations/__pycache__/
```

Chỉ cần config chung như này e nhé
```suggestion
*.pyc
__pycache__
```

---

Vâng ạ